docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
```

Comment:
Cool, I think we should also link this file from the main README

Reply:
Already done.